### ISSUE TITLE
Auth: If auth failed show error message to user with fail reason

### ISSUE THREAD

**Issue by edda** [sapcc/elektra#388]
If user/pw authentication/authorization fails the user doesn't see any error messages currently. We should display an error message with as much information as we can get (api error message or failing that the auth step that failed, etc.).

For example if a user is disabled it would help immensely if the user sees that that is the reason they can't log in.

**Comment by edda** [comment]
auth gem -> sessions_controller

**Comment by edda** [comment]
Got a new use case for this: rate limiting. We get the following response from `auth/domains`:

```
14:01:55 rails.1   | <- Net::HTTPTooManyRequests
14:01:55 rails.1   | <- body:  {"error": {"status": "429 Too Many Requests", "message": "Too Many Requests"}}
```

The user sees a blank screen with http 200.

**Comment by andypf** [comment]
fixed with d3bc932e35544ded37ad3c05d9347b5e4797e533
